dotCMS/src/main/java/com/dotcms/rest/api/v1/page/PageContainerForm.java

```diff
@@ -0,0 +1,94 @@
+package com.dotcms.rest.api.v1.page;
+
+import com.dotcms.repackage.com.fasterxml.jackson.core.JsonParser;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.DeserializationContext;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.JsonDeserializer;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.JsonNode;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+
+/**
+ * {@link PageResource#addContent(HttpServletRequest, HttpServletResponse, String)}'s form
+ */
+@JsonDeserialize(using = PageContainerForm.ContainerDeserialize.class)
+public class PageContainerForm {
+
+    private final List<ContainerEntry> entries;
+    private final String requestJson;
+
+    public PageContainerForm(final List<ContainerEntry> entries, final String requestJson) {
+        this.entries = entries;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Store a copy of "entries". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2384)